examples/quickstart/src/main/java/quickstart/ReadmeSnippets.java

```diff
@@ -159,10 +159,10 @@ private void createSwaApplication() {
         SwaApplication swaApp = client.instantiate(SwaApplication.class)
             .setSettings(client.instantiate(SwaApplicationSettings.class)
             .setApp(client.instantiate(SwaApplicationSettingsApplication.class)
-              .setButtonField("btn-login")
```

Comment:
revert this change, those incidents should be there, as this is a nested object

Reply:
don e